fusion/Navbar.js

```diff
@@ -37,31 +66,54 @@ const anchor = css`
   text-transform:
 `;
 
-const Navbar = ({ NavList }) => (<NavbarDiv>
-  <div className={lists}>
-    {!!NavList && NavList.map(({ link, navName } , i) => (<div key={i} className={links}>
-      <a className={anchor} key={link} href={link}>{navName}</a>
-    </div>))}
-  </div>
-</NavbarDiv>);
+const Navbar = ({ NavList }) => {
+  var toRender;
```

Comment:
created a ul, li list for navbar and whatever menu have sub-menu that will be passed as "submenu" props which is iterated again ,depending on whether submenu is present or not different css is also applied on it